script.js

```diff
@@ -1,4 +1,73 @@
+var playerOneGuess = "";
+var playerTwoGuess = "";
+var diceRollOne = 0;
+var diceRollTwo = 0;
+var playerOneNumber = 0;
+var playerTwoNumber = 0;
+var gameState = "";
+var myOutputValue = "";
+
+//dice roll function
+var diceRoll = function () {
+  var oneToFive = Math.floor(Math.random() * 6);
+  var randomNumber = oneToFive + 1;
+  return randomNumber;
+};
+//Helper function for selecting combination which gives largest value
+var playerOneCombinationGenerator = function () {
+  diceRollOne = diceRoll();
+  diceRollTwo = diceRoll();
```

Comment:
This could be in its own helper function to make playerOneCombinationGenerator() shorter and more concise

Reply:
I'm so sorry but I'm not too sure what you mean by this. What would be the purpose of the new helper function? To just generate two dice rolls?

Reply:
This may not be the best example since it's just two lines, but typically when there is code that repeats, we want to put them into a helper function. 

One basic coding principle is to keep your code DRY: Don't Repeat Yourself. And since you're gonna roll two dice at lines 18-19 and again at lines 34-35, we would rather put it into a helper function and call the helper function here.

Imagine having 10 players instead of just 2? That would be an additional 10 lines of code. Now image if there was three other areas that you could have refactored into a helper function? The code would have more lines than necessary and would be less readable than it could potentially be.

Hope this helps!